_assets/stylesheets/app.scss

```diff
@@ -1,8 +1,9 @@
 /*
 *= require poole
 *= require syntax
-*= require hyde 
```

Comment:
No worries, removed extra whitespace. Your editor may automagically do that for you.